app/components/AppHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import { Link } from '@remix-run/react';
+import { AppHeaderLink } from './AppHeaderLink';
+import { useAuth } from 'hooks/useAuth';
```

Comment:
다른 곳을 보니 절대경로를 사용하고 있어서 절대경로로 통일하면 어떨까요? 나중에 이 파일(`app/components/AppHeader.tsx`)의 디렉토리가 바뀌더라도 이 파일의 import문들은 수정할 필요가 없어져서 편할 듯합니다!
```suggestion
import { AppHeaderLink } from '~/components/AppHeaderLink';
import { useAuth } from '~/hooks/useAuth';
```